Formula/o/openvino.rb

```diff
@@ -128,10 +140,11 @@ def install
       -DENABLE_SYSTEM_PROTOBUF=ON
       -DENABLE_SYSTEM_FLATBUFFERS=ON
       -DENABLE_SYSTEM_SNAPPY=ON
+      -DProtobuf_USE_STATIC_LIBS=OFF
```

Comment:
I recall that the contributor of this formula was adamant that Protobuf must be linked statically (though that isn't really viable anymore for versions that link to Abseil) -- has that changed/been fixed?

Reply:
Not sure what exact restriction was so can't comment on that. The Conda recipe seems to allow it https://github.com/conda-forge/openvino-feedstock/commit/95cc49df58259251d838829da33755661388ca89

Reply:
Seems fine then.